model/server.yaml

```diff
@@ -16,38 +16,11 @@ groups:
         note: |
           When observed from the client side, and when communicating through an intermediary, `server.address` SHOULD represent
           the server address behind any intermediaries (e.g. proxies) if it's available.
-        examples: ['example.com']
+        examples: ['example.com', '10.1.2.80', '/tmp/my.sock']
       - id: port
         type: int
-        brief: Server port number
+        brief: Server port number.
         note: >
           When observed from the client side, and when communicating through an intermediary, `server.port` SHOULD represent
           the server port behind any intermediaries (e.g. proxies) if it's available.
         examples: [80, 8080, 443]
-      - id: socket.domain
```

Comment:
should we remove `domain` attribute from `source` and `destination` namespaces for now as well?

Reply:
done in this PR for consistency